.github/workflows/publishbot.yml

```diff
@@ -0,0 +1,46 @@
+name: Publish Robot
+on:
+  push:
+    branches:
+      - master
+    tags: ['*']
+jobs:
+  build:
+    name: "Build"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: ./.github/actions/setup
+  npm-publish:
+    name: 'Publish to NPM'
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: ./.github/actions/setup
+      - run: npm run npm:publish:alpha
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+  deploy:
+    name: 'Deploy the package'
+    needs: build
+    runs-on: [self-hosted, linux, x64, ec2-alpha]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: ./.github/actions/setup
+      - run: rm -rf veracode && mkdir veracode
+      - run: mkdir veracode/auth
+      - run: cp -R packages/auth/src packages/auth/package.json packages/auth/package-lock.json veracode/auth
+      - run: mkdir veracode/bueno
+      - run: cp -R packages/bueno/src packages/bueno/package.json packages/bueno/package-lock.json veracode/bueno
+      - run: mkdir veracode/headless
+      - run: cp -R packages/headless/src packages/headless/package.json packages/headless/package-lock.json veracode/headless
+      - run: mkdir veracode/atomic
+      - run: cp -R packages/atomic/src packages/atomic/package.json packages/atomic/package-lock.json veracode/atomic
```

Comment:
We could:
* create the veracode package here (current solution)
* extract it into a dedicated action
* move it into `create-deployment-package`

I don't feel strongly about any of those.